cli/pcluster/utils.py

```diff
@@ -771,6 +771,19 @@ def get_info_for_amis(ami_ids):
         error(e.response.get("Error").get("Message"))
 
 
+def validate_pcluster_version_based_on_ami_name(ami_name):
+    match = re.match(r"(.*)aws-parallelcluster-(\d+\.\d+\.\d+)(.*)", ami_name)
```

Comment:
what happens if the customer provides an AMI that is not one of the ones we publish but has a similar pattern in the name? Is there a way to make this check more robust?